incubator/command-config/src/main/moditect/module-info.java

```diff
@@ -27,6 +27,8 @@
 
     opens io.aklivity.zilla.runtime.command.config.internal.openapi.model;
 
+    opens io.aklivity.zilla.runtime.command.config.internal.asyncapi.model;
+
```

Comment:
This is needed to support reflection for the parsing, right?

Reply:
Yep, same situation as with the `*.openapi.model` that we discussed some time back... The json parser needs the access and I couldn't figure out how to narrow down the scope of `opens`.